x/bank/keeper/migrations.go

```diff
@@ -31,5 +32,10 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 
 // Migrate3to4 migrates x/bank storage from version 3 to 4.
 func (m Migrator) Migrate3to4(ctx sdk.Context) error {
-	return v4.MigrateStore(ctx, m.keeper.storeKey, m.legacySubspace, m.keeper.cdc)
+	var oldParams types.Params
+	m.legacySubspace.GetParamSet(ctx, &oldParams)
```

Comment:
I think we can call `m.legacySubspace.Get(ctx, KeySendEnabled, &sendEnabled)` directly, without changing the other part the code.

Reply:
can skip validation, since the state in db is already validated before the insertion.